src/lib/dev-environment/dev-environment-cli.js

```diff
@@ -334,14 +335,53 @@ export function addDevEnvConfigurationOptions( command ) {
 		.option( 'media-redirect-domain', 'Domain to redirect for missing media files. This can be used to still have images without the need to import them locally.' );
 }
 
-function getWordpressImageTags(): string[] {
-	return [
-		'5.9',
-		'5.8.3',
-		'5.8.2',
-		'5.8.1',
-		'5.8',
-		'5.7.3',
-		'5.7.2',
-	];
+async function populateWordPressVersionList( versionList ) {
+	const apiOptions = {
+		hostname: 'raw.githubusercontent.com',
+		path: '/Automattic/vip-container-images/master/wordpress/versions.json',
+		method: 'GET',
+	};
+
+	return new Promise( resolve => {
+		const https = require( 'https' );
```

Comment:
What about importing the `https` module at the top of the file, with the rest of the imports?